packages/lib-user/README.md

```diff
@@ -0,0 +1,49 @@
+# Zooniverse User and User Group Library
+
+A standalone library for the Zooniverse user stats, user group stats, and home pages.
+
+<!-- ## Getting Started
+
+Install the package from NPM:
+
+```sh
+npm i @zooniverse/user
+```
+
+and use it
+
+```sh
+import { UserStats } from '@zooniverse/user';
+``` -->
+
+## Run
+
+### Node/yarn
+```sh
+yarn dev
+# yarn storybook
+```
+
+Starts a development server on port 8080 ~~and a Storybook on port 6006~~ by default.
+
+Use `yarn dev` to run a small development environment app at `localhost:8080`.
+
+- a staging user stats page can be loaded by query param: `localhost:8080?users=[login]/stats`
+- a staging user group stats page can be loaded by query param: `localhost:8080?groups=[user group ID]`
```

Comment:
```suggestion
- a staging user stats page can be loaded by query param: `https://localhost:8080?users=[login]/stats`
- a staging user group stats page can be loaded by query param: `https://localhost:8080?groups=[user group ID]`
```

Suggestion to include https because auth will not work without it.

Reply:
Query params are client side only, so you won’t be able to use SSR for the user groups here.

Reply:
Having said that, we use middleware in the project app to map query params to actual paths on disk, so you could SSR these pages with some URL rewrites for page requests. 

Eg. For projects this path + query

`/projects/owner/project-name?env=staging`

is rewritten to this path before it is rendered by the Node app

`/projects/staging/owner/project-name`

Reply:
Sorry I think the readme should be updated for clarity to reflect that all the noted localhost urls with query params are for local development only, the actual url for a user group stats page, served through app-root, will be something like `https://www.zooniverse.org/groups/456789`. I'm using query params and with the local dev server to avoid using a routing library with the dev app (I think similar to lib-classifier?).

Reply:
For local dev, you'll need to run a small Next.js app. Otherwise, I don't think you'll be able to link pages with `next/link`. If you link pages with standard HTML links, then your app won't use Next's routing in the browser.